### PR TITLE
Fix Pb PDG and automatic energy setting for Pb and HI pythia configuration

### PR DESCRIPTION
**Pull request by gconesab**
Also, add argument for particle detector acceptance filter.

### CODE REVIEW

**Comment by preghenella** [comment]
@gconesab careful with p-Pb and Pb-p settings.
you cannot have a CMS energy there because of asymmetric beam energies.

best would be to redefine everything in terms of the beam energy rather than the CMS energy

File: MC/config/common/pythia8/utils/mkpy8cfg.py [hunk 1]
```diff
@@ -12,13 +12,19 @@
...
-                   help='PDG code of projectile beam')
+                   help='PDG code of projectile beam A')
 
 parser.add_argument('--idB', type=int, default='2212',
-                   help='PDG code of target beam')
+                   help='PDG code of target beam B')
 
-parser.add_argument('--eCM', type=float, default='13000.',
-                    help='Centre-of-mass energy')
+parser.add_argument('--eBeamA', type=float, default='6499.',
```
Comment (preghenella) [inline MC/config/common/pythia8/utils/mkpy8cfg.py:20]: I suggest you keep the same name as the one used by Pythia, namely `--eA` and `--eB`

File: MC/config/common/pythia8/utils/mkpy8cfg.py [hunk 2]
```diff
@@ -12,13 +12,19 @@
...
+                    help='Energy of beam A')
+
+parser.add_argument('--eBeamB', type=float, default='6499.',
```
Comment (preghenella) [inline MC/config/common/pythia8/utils/mkpy8cfg.py:23]: same here

**Comment by gconesab** [comment]
I will merge this today unless I get more comments.

**Pull request merged**